app/src/main/java/org/systers/mentorship/remote/BaseUrl.kt

```diff
@@ -21,7 +21,9 @@ object BaseUrl {
 
     val apiBaseUrl: String
         get() = if (BuildConfig.BUILD_TYPE == "debug"){
-            "$PROTOCOL_HTTPS$DEVELOPMENT_URL$EB_REGION"
+            // "$PROTOCOL_HTTPS$DEVELOPMENT_URL$EB_REGION"
+            // TODO: once backend is fixed remore this temporary URL
```

Comment:
```suggestion
            // TODO: once backend is fixed remove this temporary URL
```
Detected by @atm1504 ;)